src/pages/profile/Profile.jsx

```diff
@@ -0,0 +1,186 @@
+                <td>{email}</td>
+              </tr>
+              <tr>
+                <td>Location</td>
+                <td>Pune</td>
+              </tr>
+            </tbody>
+          </table>
+          <button
+            className="button danger logout-btn mt-1"
+            onClick={logOutHandler}
+          >
+            Logout
+          </button>
+        </section>
+
+        <section
+          className={
+            showSection === "address-section"
+              ? "address-section display-block"
+              : "address-section"
+          }
+        >
+          <div className="flex-column items-center gap-2">
+            <div className="address-title items-center">
+              <h6 className="m-0">Saved Addresses</h6>
+              <button className="button inverted-secondary radius-0">
+                Add New Address
+              </button>
+            </div>
+            <div className="card-horizontal radius-0 border-1">
+              <div className="address-header border-bottom-1 font-size-md p-0p5 px-1">
+                <span>Apurva Sawant</span>
+                <span className="text-xs basic-bg radius-4px px-0p5">Home</span>
+              </div>
+              <p className="description m-0 p-1">
+                Shop No. 3 and 4, Sohrab Hall, Opp. Jehangir Hospital, Mahatma
+                Gandhi Rd, Camp. Pune, Maharashtra, 411001
+                <span className="display-block pt-0p5 pb-1">
+                  Mobile: 9800980098
+                </span>
+                <label htmlFor="default-address">
+                  <input
+                    id="default-address"
+                    className="checkbox-field"
+                    type="checkbox"
+                  />{" "}
+                  Default Address
+                </label>
+              </p>
+
+              <div className="action-buttons border-top-1">
+                <span className="button-link info-text border-right-1 p-0p5 edit-btn">
+                  Edit
+                </span>
+                <span className="button-link danger-text p-0p5 remove-btn">
+                  Remove
+                </span>
+              </div>
+            </div>
+
+            <div className="card-horizontal radius-0 border-1">
+              <div className="address-header border-bottom-1 font-size-md p-0p5 px-1">
+                <span>Apurva Sawant</span>
+                <span className="text-xs basic-bg radius-4px px-0p5">Work</span>
+              </div>
+              <p className="description m-0 p-1">
+                MIDC, Mira Road East, Mira Bhayandar, Maharashtra, 401107
+                <span className="display-block pt-0p5 pb-1">
+                  Phone: 1800 0000 000
+                </span>
+                <label htmlFor="default-address2">
+                  <input
+                    id="default-address2"
+                    className="checkbox-field"
+                    type="checkbox"
+                  />{" "}
```

Comment:
remove these extra curly braces.

Reply:
It's there to add space between the checkbox and label